didx-xyz/yoma-mobile
# YOMA-522 - feat(Education): creating widget with real data

## Scope [YOMA-522](https://yomaworld.atlassian.net/browse/YOMA-522)
- setup the Education widget with real data

![](https://media.giphy.com/media/Q8rwlNTcDAU3MuUzd7/giphy-downsized.gif?cid=ecf05e4738h6puj5d6rql2u8vj91gnki20keue4nm6c2k3fi&rid=giphy-downsized.gif&ct=g)

## Work Done
- adding UserQualifications selector
- adding Experience Widget with selector to get data for the widget
- had to add some fixture data for qualifications

## Steps to Test
- run tests
- make sure the app looks ok
    
## Screenshots
